webpack/assets/javascripts/react_app/redux/actions/hosts/powerStatus/index.js

```diff
@@ -1,9 +1,11 @@
 import { HOST_POWER_STATUS } from '../../../consts';
 import { API_OPERATIONS } from '../../../API';
+import { getIDFormatters } from '../../../API/APIHelpers';
 
 export const getHostPowerState = ({ id, url }) => ({
   type: API_OPERATIONS.GET,
   key: HOST_POWER_STATUS,
   url,
   payload: { id },
+  ...getIDFormatters(),
```

Comment:
I'm getting a console warning because the error is set to null in success and request.
Warning: Failed prop type: Invalid prop `error` of type `object` supplied to `PowerStatusInner`, expected `string`.

Maybe change the formatter?

Reply:
Are you sure? I couldn't reproduce, see the state error is null:
![Screenshot from 2019-12-12 12-11-27](https://user-images.githubusercontent.com/26363699/70703330-8d752600-1cd8-11ea-913c-141d72a39906.png)

Reply:
thanks, I found it on failure state :+1: fixed